grails-app/controllers/au/org/ala/biocollect/RecordController.groovy

```diff
@@ -21,6 +30,8 @@ class RecordController {
         }
         model.records = results?.list
         model.total = results?.total
+        String userId = userService.getCurrentUserId()
+        model.showCrud = userId && id ? projectService.isUserAdminForProject(userId, id) : false
```

Comment:
Does this mean we only show the add/edit/delete options for project admins? Should also be for the owner of the activity/record, shouldn't it?